### PR TITLE
Landingpage

### PR DESCRIPTION
**Pull request by JonathanYesh279**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,39 +1,25 @@
...
-                  </Routes>
-                </div>
-              </div>
-            </main>
-          </div>
+        <Routes>
+          <Route path='/' element={<Navigate to='/home' replace />} />
+          <Route path='/home' element={<HomePage />} />
+          {/* <Route path='/board/*' element={<Board />} /> */}
+        </Routes>
```
Comment (tomercarolla) [inline src/App.jsx:21]: please fix the routing. add route to login and another route to board, maybe need to add a Layout.jsx component and there do a routing for the board.

File: src/assets/styles/pages/homepage.scss [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+ box-sizing: border-box;
+ font-family: 'Poppins', sans-serif;
+}
+
+.header {
+  position: fixed;
+  inset: 0 0 auto 0;
+  font-weight: 600; 
+  display: flex;
+  z-index: 1000;
```
Comment (tomercarolla) [inline src/assets/styles/pages/homepage.scss:14]: dont think you need z index here

File: src/assets/styles/pages/homepage.scss [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+ width: 100%;
+ height: 100vh;
+ overflow: hidden;
+ box-sizing: border-box;
+ font-family: 'Poppins', sans-serif;
+}
+
+.header {
+  position: fixed;
+  inset: 0 0 auto 0;
```
Comment (tomercarolla) [inline src/assets/styles/pages/homepage.scss:11]: inset: 0; is enough it means top bottom right and left 0

File: src/assets/styles/pages/homepage.scss [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+  font-weight: 600; 
+  display: flex;
+  z-index: 1000;
+  
+
+
+  .home-nav {
+   display: flex;
+   align-self: center;
+   -webkit-box-align: stretch;
```
Comment (tomercarolla) [inline src/assets/styles/pages/homepage.scss:21]: remove web-kit

File: src/assets/styles/pages/homepage.scss [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+  background-position: center calc(100% + 120px), 0 0;
+  background-repeat: no-repeat, no-repeat;
+  background-size: 100% auto, cover;
+  color: #fff;
+  height: 67%;
+
+    .bg {
+      padding-top: 10%;
+      margin-left: auto;
+      margin-right: auto;
```
Comment (tomercarolla) [inline src/assets/styles/pages/homepage.scss:89]: margin-inline: auto;

File: src/assets/styles/pages/homepage.scss [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+      padding-left: 1rem;
+      padding-right: 1rem;
```
Comment (tomercarolla) [inline src/assets/styles/pages/homepage.scss:91]: padding-inline: 1rem; if you copy from trello please check what is the value in px.

File: src/assets/styles/pages/homepage.scss [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+
+      .content {
+        -webkit-box-align: center;
+        align-items: center;
+        -webkit-box-pack: start;
+        justify-content: flex-start;
+        display: flex;
+        flex-wrap: wrap;
+        margin-left: -1rem;
+        margin-right: -1rem;
```
Comment (tomercarolla) [inline src/assets/styles/pages/homepage.scss:103]: stop copying from trello, remove the web-kit
margin-inline: -1rem; please check the value in px and why negative value, maybe has another way.

File: src/assets/styles/pages/homepage.scss [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+              border-color: #0065ff;
+              box-shadow: 0 0 0 3px rgba(0, 101, 255, 0.3);
+              transform: translateY(-2px);  
+            }
+
+          }
+
+          .email-button {
+            appearance: none;
+            -webkit-box-align: center;
```
Comment (tomercarolla) [inline src/assets/styles/pages/homepage.scss:140]: remove web-kit

File: src/ui/Icons/png-icons/HomeImage.png
Comment (tomercarolla) [inline src/ui/Icons/png-icons/HomeImage.png:1]: this image is just in homepage. move to homepage folder. add in homepage folder assets folder and move there.

File: src/ui/Icons/png-icons/wave.svg
Comment (tomercarolla) [inline src/ui/Icons/png-icons/wave.svg:1]: this image is just in homepage. move to homepage folder. add in homepage folder assets folder and move there.

File: src/assets/styles/pages/homepage.scss (hunk 3 above)
Comment (JonathanYesh279) [reply src/assets/styles/pages/homepage.scss:11]: It doesn't work. I tried some combinations, and this one straightened it to the middle

File: src/pages/HomePage.jsx [hunk 9]
```diff
@@ -1,9 +1,71 @@
...
+                  <input
+                    className='email-input'
+                    type='email'
+                    placeholder='Enter your email'
+                  />
+                  <button className='email-button'>Sign up it's free!</button>
+                </form>
+              </div>
+              <pciture className='right'>
+                <img src='./src/ui/Icons/png-icons/HomeImage.png' />
```
Comment (tomercarolla) [inline src/pages/HomePage.jsx:63]: Move the png to assets folder inside homepage folder. 
inside pages folder create homepage folder and move there the homepage.jsx, also create assets folder and add there the images. REMOVE THEM FROM ICONS folder

File: src/ui/Icons/svg-icons/icons.jsx [hunk 10]
```diff
@@ -52,5 +55,7 @@ export const icons = {
     bell: Bell,
     questionMark: QuestionMark,
     trash: Trash,
+    trello: Trello,
+    wave: Wave,
```
Comment (tomercarolla) [inline src/ui/Icons/svg-icons/icons.jsx:59]: remove the wave from here, its not a icon its a image. move this to pages/homepage/assets folder

**Review by tomercarolla — APPROVED** [review]

**Pull request merged**